doc/design/embargo-mvp.md

```diff
@@ -0,0 +1,131 @@
+# Embargo Minimum Viable Product
+Due to time constraints, we are implementing the core functionality for embargoing dandisets ASAP.
+This document is dedicated to the minimum viable product.
+A subsequent document will describe the full suite of features.
+
+
+# User experience
+
+## Web
+When creating a new dandiset, users will have checkbox to choose whether or not the dandiset should be embargoed.
+If so, the user must also specify an NIH award number (https://era.nih.gov/files/Deciphering_NIH_Application.pdf).
+The award number is an optional argument that is passed to the create dandiset endpoint.
+The award number will be added as a new `Funder` on an automatically generated Contributor Organization in the metadata.
+
+Instead of the normal `Publish` button and version browser, the DLP will instead show a `Release` button.
+Clicking it will open a confirmation modal informing the user that they are unembargoing, all their data will be publicized, there is no undoing, and that it can take some time for large dandisets.
+Confirming will lock the dandiset for the duration of the release.
+Once the release finishes, the dandiset will be like any other unpublished draft dandiset.
+
+Once created, an embargoed dandiset is only visible or searchable to owners.
+Instead of the `draft` chip, an `embargoed` chip should be used instead.
```

Comment:
when you say chip here, i'm assuming you mean ui. will the api endpoint version parameter say `embargo` instead of `draft` or will it remain as `draft` with simply the metadata containing access info. i think i would prefer to continue using "draft" as a version identifier for this.

Reply:
We could just have an additional `embargo` chip next to the draft one.